### DISCUSSION TITLE
Mobile: portraits images full size

### DISCUSSION THREAD

**Discussion by TarasBounty** [sachinchoolur/lightGallery#1362] [Q&A]
Hello!

I'm using [Version 2.6.0-beta.1 release](https://github.com/sachinchoolur/lightGallery/releases/tag/2.6.0-beta.1) and everything works good on PC browsers.

But on mobile, portrait images doesn't fit available screen size:
![image](https://user-images.githubusercontent.com/15182848/183076710-f457faa0-14fe-42fe-bd1e-e053f2b2fcea.png)

**Comment by VadzimPapko** [comment]
Hi @sachinchoolur, getting the same issue. you might already have a fix. 
Could you please review this one?
Thanks!

**Comment by sachinchoolur** [comment]
Hey @TarasBounty and @VadzimPapko ,

It works fine for me. Is it possible to share the website URL? 

If not, please share the image dimension. I'll try to reproduce the issue from my side
